Core/GDCore/Project/EffectsContainer.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * GDevelop Core
+ * Copyright 2008-2016 Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#ifndef GDCORE_EFFECTS_CONTAINER_H
+#define GDCORE_EFFECTS_CONTAINER_H
+#include <memory>
+#include <vector>
+
+#include "GDCore/String.h"
+
+namespace gd {
+class Effect;
+}  // namespace gd
+
+namespace gd {
+
+/**
+ * \brief Contains the effect applied to other entities (ie, Layer or Object)
```

Comment:
ubernitpicking: "i.e: " ;) Also end with a period.

`Contains effects applied to an entity on screen (i.e: a Layer or an Object).`